examples/remote_execution/buildbuddy/platforms/defs.bzl

```
@@ -0,0 +1,36 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under both the MIT license found in the
+# LICENSE-MIT file in the root directory of this source tree and the Apache
+# License, Version 2.0 found in the LICENSE-APACHE file in the root directory
+# of this source tree.
+
+def _platforms(ctx):
+    configuration = ConfigurationInfo(
+        constraints = {},
+        values = {},
+    )
+
+    # This is the default image used in BuildBuddy SaaS executor.
+    # This one is Ubuntu 22.04 (i.e. its current LTS release): https://hub.docker.com/_/ubuntu
+    image = "docker://gcr.io/flame-public/rbe-ubuntu20-04:latest"
+    platform = ExecutionPlatformInfo(
+        label = ctx.label.raw_target(),
+        configuration = configuration,
+        executor_config = CommandExecutorConfig(
+            local_enabled = True,
+            remote_enabled = True,
+            use_limited_hybrid = True,
+            remote_execution_properties = {
+                "OSFamily": "Linux",
+                "container-image": image,
+            },
+            remote_execution_use_case = "buck2-default",
+            # TODO: Use output_paths
+            remote_output_paths = "strict",
```

We actually did some testing on BuildBuddy and IIRC it does support `output_paths`. Can you try that here?


```suggestion
            remote_output_paths = "output_paths",
```

---

```
!?sluongng/add-buildbuddy ~/work/facebook/buck2/examples/remote_execution/buildbuddy> PAGER= git diff
diff --git a/examples/remote_execution/buildbuddy/platforms/defs.bzl b/examples/remote_execution/buildbuddy/platforms/defs.bzl
index 41399a0cd..073528d80 100644
--- a/examples/remote_execution/buildbuddy/platforms/defs.bzl
+++ b/examples/remote_execution/buildbuddy/platforms/defs.bzl
@@ -26,8 +26,7 @@ def _platforms(ctx):
                 "container-image": image,
             },
             remote_execution_use_case = "buck2-default",
-            # TODO: Use output_paths
-            remote_output_paths = "strict",
+            remote_output_paths = "output_paths",
         ),
     )

!?sluongng/add-buildbuddy ~/work/facebook/buck2/examples/remote_execution/buildbuddy> buck2 kill && buck2 build //...
killing buckd server
Build ID: 93d14b6d-a8c5-4fce-a55f-e81a466dc7a2
RE: GRPC-SESSION-ID
Jobs completed: 15. Time elapsed: 15.8s. Cache hits: 0%. Commands: 5 (cached: 0, remote: 3, local: 2)
BUILD SUCCEEDED
```

Yup it just worked!

I am actually improving `Command.output_paths` and `ActionResult.output_symlinks` support right now on both Bazel and BuildBuddy. 🤗